recipes/boost/all/conanfile.py

```diff
@@ -1607,7 +1483,18 @@ def filter_transform_module_libraries(names):
                     new_name = add_libprefix(name.format(**libformatdata)) + libsuffix
                     if self.options.namespace != 'boost':
                         new_name = new_name.replace("boost_", str(self.options.namespace) + "_")
+
                     if name.startswith("boost_python") or name.startswith("boost_numpy"):
+                        # FIXME: these are missing python_buildid and buildid
```

Comment:
To fix this I'll probably want to rewrite stuff to put *all* pythons into a single OrderedDict.